include/deal.II/lac/lapack_full_matrix.h

```diff
@@ -746,6 +746,12 @@ class LAPACKFullMatrix : public TransposeTable<number>
    */
   void compute_inverse_svd (const double threshold = 0.);
 
+  /**
+   * Same as above but provide the size of the kernel instead of a threshold,
+   * i.e. the @p kernel_size smallest eigenvalues.
+   */
+  void compute_inverse_svd (const unsigned int kernel_size = 0);
```

Comment:
You can have two functions with the same name with default arguments? How does that work? Regardless, I don't think that is a good idea.